config.json

```diff
@@ -34,7 +34,7 @@
       "description": "MsBuild building options.",
       "valueType": "passThrough",
       "values": [],
-      "defaultValue": "/nologo /maxcpucount /verbosity:minimal"
+      "defaultValue": "/nologo /maxcpucount /verbosity:minimal /nodeReuse:false"
```

Comment:
Maybe we could have a central config.json file in the future, when we implementing the public API for the repos. I'll keep it in mind.

Thanks @joperezr